server/settings.py

```diff
@@ -166,6 +166,20 @@
     'graphic': graphic_profile['schema'],
 }
 
+SCHEMA_UPDATE = {
+    'archive': {
+        'extra': {
+            'type': 'dict',
+            'schema': {},
+            'mapping': {
+                'type': 'object',
+                'enabled': True
```

Comment:
it would be good to define the fields explicitly so there won't be any issues with auto type detection

Reply:
updated